bundle/manifests/elasticsearch-operator-alerts_monitoring.coreos.com_v1_prometheusrule.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    name: elasticsearch-operator
+  name: elasticsearch-operator-alerts
+spec:
+  groups:
+  - name: elasticsearch_operator.alerts
+    rules:
+    - alert: ElasticsearchOperatorRemovalWarning
+      annotations:
+        message: Elasticsearch Operator is deprecated and will be removed from the
```

Comment:
late remark but it should be `description` as per https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/alerting-consistency.md#style-guide
I suppose that all rules have the same issue and it has been reported in our Slack channel.

Reply:
https://issues.redhat.com/browse/LOG-6126